app/Commands/ConfigDumpCommand.php

```diff
@@ -26,29 +27,47 @@ public function handle()
     {
         if ($this->option('pint') || $this->option('all')) {
             $this->exportFiles(
-                ['pint.json'],
-                (bool) $this->option('force')
+                [
+                    'stubs/pint.json' => 'pint.json',
+                ],
+                (bool)$this->option('force')
             );
         }
 
         if ($this->option('php-cs-fixer') || $this->option('all')) {
             $this->exportFiles(
-                ['pint.json', '.php-cs-fixer.php'],
-                (bool) $this->option('force')
+                [
+                    'pint.json' => 'pint.json',
```

Comment:
should pint be here?

Reply:
yea, `.php-cs-fixer.php` imports and parses the `pint.json` file
it basically exists to help the IDE auto format